task.go

```diff
@@ -0,0 +1,71 @@
+package mesg
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	"github.com/mesg-foundation/core/api/service"
+)
+
+var errOnlyOneOutputKey = errors.New("there should be only one output key in response")
```

Comment:
Can you create a dedicated error type? We should avoid `errors.New` in all the codebase.

Reply:
done